plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/build/ProjectStatePersister.java

```diff
@@ -509,7 +509,7 @@ private URI makeAbsolute(URI baseURI, String relativeString) {
 		if (relativeString.contains(":/")) {
```

Comment:
Not part of this PR's topic, but we should consider finding some other approach here to tell apart absoulte and relative URIs.